lib/manager/terragrunt/readme.md

```diff
@@ -0,0 +1,18 @@
+Currently by default, Terragrunt support is limited to Terraform registry sources and GitHub sources that include semver refs, e.g. like `github.com/hashicorp/example?ref=v1.0.0`.
+
+You can create a custom [versioning config](../../../docs/usage/configuration-options.md#versioning) to support non-semver references.
+For example, if you want to reference a tag like `module-v1.2.5`, a block like this would work:
+
+```json
+"terraform": {
+	"versioning": "regex:^((?<compatibility>.*)-v|v*)(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)$"
+}
+```
+
+Fixed Terragrunt versions like the following will receive a PR whenever there is a newer version available:
+
+```hcl
+terraform {
+  source = "github.com/hashicorp/example?ref=v1.0.0"
```

Comment:
At line 1 in this document you say: "GitHub sources that include semver refs, e.g. like `github.com/hashicorp/example?ref=v1.0.0`."

But why do we use `source = "github.com/hashicorp/example?ref=v1.0.0"` in the "Fixed Terragrunt versions" example then? Are you sure that makes sense?

It's a bit confusing to me, to first say "Oh this works by default", and then later in a an code block say: "Oh wait you need to fix this by doing this thing".

Reply:
Sorry but I don't understand what the problem is. It's same GitHub source like in the example.

Reply:
Furthermore, people using terragrunt or terraform need to know how to use these tools before using renovate.

Reply:
Let's "resolve" this then, you're the maker of the code, so you know best if the docs are correct. :wink: 

Can you click the "resolve" button on this conversation? I don't have the rights to mark this as "resolved" myself. :smile:

Reply:
Sure thing :)